lib/benchee/benchmark/runner.ex

```diff
@@ -33,10 +33,10 @@ defmodule Benchee.Benchmark.Runner do
                           scenario_context = %ScenarioContext{printer: printer, config: config}) do
     printer.input_information(input_name, config)
     printer.benchmarking(job_name, config)
-    pmap 1..config.parallel, fn ->
+    Parallel.map(1..config.parallel, fn(_) ->
```

Comment:
Yeah, hopefully. For me I thought if I came back to this later I might think, "wait, why are we even passing the task number to this function if it's not used?!".